### PR TITLE
DG and README improvements

### PR DESCRIPTION
**Pull request by clarencepohh**
README: 
- Update AboutUs.md to reflect github pages as well as PPP links
- Create skeleton PPP files for each team member to use for submission

DG: 
- Add target user profile description
- Add 1 (only..) NFR and glossary for .ics file 
- Add value proposition for product 
- Add v1.0 and v2.0 user stories

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -463,26 +463,45 @@ The 'ics' component:
...
+| v2.0    | user         | add, remove and edit tasks with fewer keystrokes                             | manage my busy schedule more efficiently                                    |
+| v2.0    | user         | set priority levels for tasks and events                                     | can easily identify what requires immediate attention                       |
+| v2.0    | user         | have a deeper connection with the application with the addition of an avatar | feel more engaged when using the application.                               |
+| v2.0    | user         | be able to mark tasks as complete                                            | can keep track of what I have left to do.                                   |
+| v2.0    | user         | have the application look neat                                               | will be more motivated to use the application                               |
+| v2.0    | busy student | be able to see different views (week/month) on the calendar                  | can see my week/month at a glance                                           |
```
Comment (NicholasTanYY) [inline docs/DeveloperGuide.md:494]: Great user stories!

**Review by NicholasTanYY — APPROVED** [review]
LGTM. This is a great starting point towards the completion of a comprehensive DG.

File: build.gradle [hunk 2]
```diff
@@ -37,6 +37,7 @@ application {
 shadowJar {
     archiveBaseName.set("duke")
     archiveClassifier.set("")
+    archiveFileName = 'clinton.jar'
```
Comment (Mahesh1772) [inline build.gradle:40]: Good job to rename our jar to match the application!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -463,26 +463,45 @@ The 'ics' component:
 ## Product scope
 ### Target user profile
 
-{Describe the target user profile}
+Our target users are those require a calendar or task-management application but do not like clicking through multiple window prompts just to do so.
```
Comment (Mahesh1772) [inline docs/DeveloperGuide.md:466]: Great job describing the perfect user profile!

File: docs/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # CLI-nton
 
-{Give product intro here}
+Welcome to CLI-nton, a powerful task management application designed to help you stay organized and productive. With a simple command-line interface, CLI-nton makes it easy to manage your tasks, whether they're todos, deadlines, or events.
```
Comment (Mahesh1772) [inline docs/README.md:3]: Good to have a introductory line

**Review by Mahesh1772 — COMMENTED** [review]
Overall LGTM, good work on the documentation

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -463,26 +463,45 @@ The 'ics' component:
...
+have to navigate through many windows just to add one event to their calendar. This application is made to benefit those who type quickly and accurately, allowing them to quickly create entries
+in their calendar. 
 
 ## User Stories
 
-| Version | As a ... | I want to ...             | So that I can ...                                           |
-|---------|----------|---------------------------|-------------------------------------------------------------|
-| v1.0    | new user | see usage instructions    | refer to them when I forget how to use the application      |
-| v2.0    | user     | find a to-do item by name | locate a to-do without having to go through the entire list |
+| Version | As a ...     | I want to ...                                                                | So that I ...                                                               |
```
Comment (kyhjonathan) [inline docs/DeveloperGuide.md:479]: Good job on adding the user stories!

**Pull request merged**